src/components/Settings/RecorderSettings.tsx

```diff
@@ -57,7 +57,7 @@ export const RecorderSettings = () => {
         name="recorder.browserPath"
         onSelectFile={handleSelectFile}
         buttonText="Select executable"
-        hint="The location of the browser executable (Grafana k6 Studio currently supports Chrome)"
+        hint="The location of the browser executable (Google Chrome needs to be installed on your machine for the recording functionality to work)"
```

Comment:
We're testing Linux with arm64, which doesn't support Chrome but uses Chromium instead. At some point we'll have to conditionally change the browser name here.

https://github.com/grafana/k6-studio/pull/513/files#diff-e84e938a5120632cd9bfda48f378416139afdb46dbb35444370bae86a6f1c248R25-R27

Reply:
IMO we either officially support Chromium or we don't and looks like we need to officially support it, in which case it should say `Chromium or Google Chrome` as one shouldn't have to have Chrome installed on Mac either